src/utils/conversion.ts

```diff
@@ -3783,17 +3784,18 @@ export function sortTransferRestrictionByClaimValue(
   const getJurisdictionValue = (
     condition: PolymeshPrimitivesTransferComplianceTransferCondition
   ): Option<MeshCountryCode> | undefined => {
-    if (condition.isClaimCount) {
-      if (!condition.asClaimCount[0].isJurisdiction) {
+    const { isClaimCount, asClaimCount, isClaimOwnership, asClaimOwnership } = condition;
```

Comment:
You can't destructure the `asX` members right away, they are getters that throw if the `isX` part is false. You have to destructure them inside the corresponding if